File: app/src/main/java/com/rayworks/droidweekly/model/ArticleItem.java
package com.rayworks.droidweekly.model;

public class ArticleItem {
    public String title;

    public String description;

    public String linkage;

    public String imageUrl;

    public int imgFrameColor;

    public ArticleItem(String title, String description, String linkage) {
        this.title = title;
        this.description = description;
        this.linkage = linkage;
    }

    public ArticleItem() {}

    @Override
    public String toString() {
        return "ArticleItem{"
                + "title='"
                + title
                + '\''
                + ", description='"
                + description
                + '\''
                + ", linkage='"
                + linkage
                + '\''
                + ", imageUrl='"
                + imageUrl
                + '\''
                + ", imgFrameColor="
                + imgFrameColor
                + '}';
    }
}

File: app/src/main/kotlin/com/rayworks/droidweekly/model/ArticleItem.kt
package com.rayworks.droidweekly.model

/***
 * The view data for an [Article]
 */
data class ArticleItem(var title: String = "", var description: String = "",
                       var linkage: String = "") {

    var imageUrl: String? = null

    var imgFrameColor: Int = 0
}
